exes/ethercat/inc/public/tfc/ec/config/bus.hpp

```diff
@@ -39,11 +40,15 @@ struct tfc::json::detail::to_json_schema<tfc::ec::config::network_interface> {
 };
 
 namespace tfc::ec::config {
-struct bus {
+struct ethercat {
   network_interface primary_interface{ tfc::global::get_interfaces()[0] };
+  confman::observable<std::optional<std::size_t>> minimum_slave_count{ 0 };
```

Comment:
```suggestion
  confman::observable<std::optional<std::size_t>> required_slave_count{ 0 };
```

I don't see a use case for the minimum amount. Should we not just exact match?

Reply:
sounds good, changing